effekt/jvm/src/main/scala/effekt/Repl.scala

```diff
@@ -108,7 +108,7 @@ class Repl(driver: Driver) extends REPL[Tree, EffektConfig, EffektError] {
       runFrontend(StringSource(""), module.make(UnitLit()), config) { cu =>
         module.definitions.foreach {
           case u: Def =>
-            outputCode(DeclPrinter(context.symbolOf(u)), config)
+            outputCode(DeclPrinter(context.symbolsOf(u)), config)
```

Comment:
My intuition tells me that there are invariants which are not represented here. This will lead to *a lot* of code changes that might not be necessary.

Reply:
Yes, a definition now has multiple symbols.